packages/cli/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@zowe/cli",
-  "version": "6.37.7",
+  "version": "6.37.8",
```

Comment:
Please revert the manual change to version number in package.json file, our automation will handle bumping it as needed.